src/explore-education-statistics-frontend/src/modules/find-statistics/components/DynamicDataBlockTabs.tsx

```diff
@@ -0,0 +1,21 @@
+import { DataBlockTabsProps } from '@common/modules/find-statistics/components/DataBlockTabs';
+import dynamic from 'next/dynamic';
+import React, { memo } from 'react';
+
+const DataBlockTabInternal = dynamic(
+  () => import('@common/modules/find-statistics/components/DataBlockTabs'),
+  {
+    ssr: false,
+  },
+);
+
+/**
+ * Wrapper component so that we can lazily render {@see DataBlockTabs}.
+ * We need this as it otherwise causes issues with SSR due to Leaflet
+ * trying to initialise server-side (causing undefined `window` errors).
+ */
+const DynamicDataBlockTab = (props: DataBlockTabsProps) => {
+  return <DataBlockTabInternal {...props} />;
+};
+
+export default memo(DynamicDataBlockTab);
```

Comment:
Rather than call this `DynamicDataBlockTab` (note it should also be plural too i.e. 'tabs'), would suggest just keeping this as it's original name of `DataBlockTabs`.

There's no need to convey the implementation detail in the naming, and it's not really clear to a person just skimming the code what 'dynamic' is supposed to mean anyway.

This should have a tiny benefit of making fewer overall changes in this PR too.

Reply:
I'm not super keen on having two components named identically, but I do agree the word dynamic is misleading. I can't think of anything better that clearly denotes the intention of this component so have gone with your suggestion :)